nodemon.json

```diff
@@ -0,0 +1,6 @@
+{
+  "watch": ["src"],
+  "ext": "ts",
+  "ignore": ["src/**/*.spec.ts"],
+  "exec": "ts-node-dev -r tsconfig-paths/register src/index.ts"
```

Comment:
Would have loved to see the inclusion of the `--transpileOnly` flag which according to documentation of ts-node-dev [here](https://www.npmjs.com/package/ts-node-dev) speeds up things greatly.  You can also include the `--inspect --` flag to enable chrome debugging in dev mode

Reply:
@Allan690 , I have made the changes and included the flags.